src/Redux/Books/Books.js

```diff
@@ -0,0 +1,22 @@
+const ADD_BOOK = 'BOOKSTORE/Books/ADD_BOOK';
+const REMOVE_BOOK = 'BOOKSTORE/Books/REMOVE_BOOK';
```

Comment:
Great job following the [Duck Pattern](https://github.com/erikras/ducks-modular-redux) 👍🏽